docs/spec/faq.md

```diff
@@ -50,3 +50,24 @@ Updates for the given subscription can then be sent from the GraphQL server to t
 To unsubscribe, the client just calls `EventSource.close()`.
 
 Also, Mercure can easily be integrated with Apollo GraphQL by creating [a dedicated transport](https://github.com/apollographql/graphql-subscriptions).
+
+## How to send authorization to Mercure?
+
+Cookies are automatically sent by the browsers when opening an `EventSource` connection if the `withCredentials` attribute is set to `true`:
+
+```javascript
+const eventSource = new EventSource(hub, {
+    withCredentials: true
+});
+```
+
+If the client is not a web browser, cookies can't be used.
+In order to achieve that, use a [polyfill](https://github.com/Yaffle/EventSource):
```

Comment:
Actually the polyfill is only useful for very old browsers and for cross-domain authorization.